### PR TITLE
build(recipe): add recipe for conda-forge

### PR DESCRIPTION
**Pull request by deepcharles**
Checklist
* [x] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [x] Bumped the build number (if the version is unchanged)
* [x] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [x] Ensured the license file is being packaged.

I also added two maintainers for this recipe (@oboulant and myself).

### CODE REVIEW

**Comment by deepcharles** [comment]
@conda-forge-admin, please rerender

**Comment by ogrisel** [comment]
As the maintainer of the upstream package, you should probably add yourself to the `recipe-maintainers` section of the `meta.yaml` file. This way you will be able to merge your own maintenance PRs on this repo.

**Comment by deepcharles** [comment]
@conda-forge-admin, please rerender

**Comment by oboulant** [comment]
I am willing to be listed here 🙂

**Comment by deepcharles** [comment]
@oboulant I added you already (in `recipe/meta.yaml`).

**Comment by oboulant** [comment]
@deepcharles I know but I just confirmed it. In the docs for the recipe creation they list as an item to post a comment of the maintainers mentioned. Since it is an addition here, I thought it might be required to have consent. So, to clear the doubt I posted the comment :)

**Comment by oboulant** [comment]
@conda-forge-admin We have a PR passing all the checks and we were wondering what would be the next steps for the PR to be merged ?
Thanks a lot again !

File: recipe/meta.yaml
Comment (xhochy) [inline recipe/meta.yaml]:
```suggestion
```

File: recipe/meta.yaml
Comment (xhochy) [inline recipe/meta.yaml]: Don't publish release candidates on the main label.

**Review by xhochy — CHANGES_REQUESTED** [review]
This is a PR for a release candidate, this should not be merged to the main label

File: recipe/meta.yaml
Comment (deepcharles) [reply recipe/meta.yaml]: Noted. Then I'll wait a few days to push the 1.1.2 version. In the meantime I have two questions:

- can you merge #8 in which I added myself and @oboulant as maintainers?
- can you point to some documentation for pushing release candidates? I have seen some discussions but nothing "official".

Comment (xhochy) [reply recipe/meta.yaml]: Have a look at https://github.com/conda-forge/cfep/blob/master/cfep-05.md

Comment (deepcharles) [reply recipe/meta.yaml]: Many thanks!

**Comment by rpanai** [comment]
> @conda-forge-admin We have a PR passing all the checks and we were wondering what would be the next steps for the PR to be merged ?
> Thanks a lot again !

I've been asked to review the PR but as @xhochy said you are trying to push a release candidate. When do you think the production release will be available?

**Comment by deepcharles** [comment]
> When do you think the production release will be available?

Next week probably.

**Comment by deepcharles** [comment]
@conda-forge-admin, please rerender

**Comment by deepcharles** [comment]
@conda-forge-admin, please rerender

**Comment by deepcharles** [comment]
This is now the correct version of `ruptures` (i.e. not a pre-release) to upload to conda-forge.

@rpanai @xhochy The PR can now be merge to master

File: recipe/meta.yaml
Comment (xhochy) [inline recipe/meta.yaml]: This is not needed, we skip Python <3.6 automatically.

File: recipe/meta.yaml
Comment (xhochy) [inline recipe/meta.yaml]: Please remove the comment

File: recipe/meta.yaml
Comment (deepcharles) [reply recipe/meta.yaml]: thanks for the quick reply. Resolved in c25ae3f.

File: recipe/meta.yaml
Comment (deepcharles) [reply recipe/meta.yaml]: resolved in c25ae3f

**Comment by deepcharles** [comment]
@conda-forge-admin, please rerender

**Review by xhochy — APPROVED** [review]

**Comment by xhochy** [comment]
@rpan PTAL. It's fine for but I would wait a bit for you before the merge.

**Review by oboulant — APPROVED** [review]
LGTM
Thanks !

**Comment by oboulant** [comment]
@rpanai Did you have any chances to have a look at this PR ? If yes, could you approve so we can move forward with the conda release of the v1.1.2 of the `ruptures` package ? 
Many thanks for the time and energy !

**Comment by rpanai** [comment]
> @rpanai Did you have any chances to have a look at this PR ? If yes, could you approve so we can move forward with the conda release of the v1.1.2 of the `ruptures` package ?
> Many thanks for the time and energy !

@oboulant Done!

**Pull request merged**
